components/sysutils/bacula/Makefile

```diff
@@ -10,39 +10,34 @@
 #
 
 #
-# Copyright 2021 Gary Mills
 # Copyright 2016 Alexander Pyhalov
+# Copyright 2021 Gary Mills
+# Copyright 2022 Friedrich Kink
 #
 
+BUILD_BITS=             64
+USE_OPENSSL11=          yes
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME= bacula
+COMPONENT_NAME=		bacula
+COMPONENT_VERSION=	11.0.5
+COMPONENT_SUMMARY=	Bacula (network backup solution)
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:ef5b3b67810442201b80dc1d47ccef77b5ed378fe1285406f3a73401b6e8111a
+COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/bacula/files/$(COMPONENT_NAME)/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://www.bacula.org
+COMPONENT_LICENSE=      AGPLv3 with exceptions
+COMPONENT_LICENSE_FILE= LICENSE
 
-COMPONENT_VERSION= 9.6.7
-COMPONENT_REVISION= 1
-COMPONENT_SUMMARY= Bacula (network backup solution)
-COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH= \
-  sha256:df0f8525ecdba380e74b537ca71adf3b7596593f02189b59ab0ebffc85ccc8de
-COMPONENT_ARCHIVE_URL= \
-  http://sourceforge.net/projects/bacula/files/bacula/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)/download
-COMPONENT_PROJECT_URL = http://www.bacula.org
+test: $(NO_TESTS)
```

Comment:
This doesn't work. Use TEST_TARGET= $(NO_TESTS) instead.

Reply:
did not check just took over but it is fixed now.